lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlMulti.pm

```diff
@@ -97,7 +98,7 @@ XML
 
     my $i=0;    
     for($i=0;$i<@bams;$i++){
-        my $path = abs_path($bams[$i]);
+        my $path = resolve_bam_path($bams[$i]);
```

Comment:
This is not very important but it looks like you could just call `resolve_bam_path` once up around [line 78][1], for example,

```
my @bams = map { resolve_bam_path ($_) } split(/\,/, $self->bams);
```

[1]: https://github.com/susannasiebert/genome/blob/igv/lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlMulti.pm#L78